src/features/entity/use_cases/validate_entity.py

```diff
@@ -95,8 +108,10 @@ def validate_entity(
         )
 
 
-def validate_entity_use_case(entity: BasicEntity, user: User) -> str:
+def validate_entity_use_case(
+    entity: BasicEntity, user: User, as_type: common_type_constrained_string | None = None
```

Comment:
No need for a default value, since that is already handled by api endpoint
```suggestion
    entity: BasicEntity, user: User, as_type: common_type_constrained_string | None
```

Reply:
fixed